vspace/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -6,7 +6,7 @@
 	<body>
 		<div th:fragment="navbar">
 			<nav class="nav-bar" style="height:48px">
-		      <a class="navbar-brand" href="#"><img th:src="@{/resources/maintenance/img/logo-11.png}" alt="Virtual Spaces logo" width="100px">
+		      <a class="navbar-brand" href="/vspace/"><img th:src="@{/resources/maintenance/img/logo-11.png}" alt="Virtual Spaces logo" width="100px">

```

Comment:
this does not work if the application isn't deployed as `vspace` which it isn't in most cases